### ISSUE TITLE
Launching devmaster server issues

### ISSUE THREAD

**Issue by jonpugh** [opendevshop/devshop.build#41]
1.0.0-beta9:
- Verify server task hangs. 
  - Cancelling that task puts the queue on hold.
  - SQL command unblocks it: "delete from semaphore;"
  - DNS does not delete records when a server is deleted.
  - incorrect DNS entries are being added:  #*bi.devshop.systems
  - SSH key variable setting still doesn't work.
  - new project from http://github.com/jonpugh/drupal8 dev environment on development ends up showing up as tag 1.0.0
  - Some bootstrap icons don't show.
  - Export config doesn't work out of the box
  - Improve size of retry task button.
  - Commit code fails. Try export config the adding the config folder.
